prow/config/templates/prow-controller-manager-Deployment.yaml

```diff
@@ -44,15 +44,20 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.name
+        - name: GITHUB_APP_ID
+          valueFrom:
+            secretKeyRef:
+              name: github-token
+              key: appid
         args:
         - --dry-run={{ .Values.dryRun }}
-        - --deck-url=http://deck/
```

Comment:
Interesting! deck is no longer needed?

Reply:
Apparently not!  peep https://raw.githubusercontent.com/kubernetes/test-infra/master/config/prow/cluster/starter/starter-s3.yaml and you will see there is no longer any mention of `deck-url`